charts/fission-core/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+name: fission
+version: 0.1.0
+description: A Helm chart for Fission. Fission is a fast serverless framework for Kubernetes.
+keywords:
+- fission
+- serverless
+home: http://fission.io/
+maintainers:
```

Comment:
@soamvasani  @life1347  You can remove my name from maintainers list. I do plan to contribute minor fixes though . 😉